examples/va/hci/service-values.yaml

```diff
@@ -37,6 +37,7 @@ data:
       store_description = "RBD backend"
       rbd_store_pool = images
       rbd_store_user = openstack
+      rbd_thin_provisioning = False
```

Comment:
the default value is False, if it is set to True then you will get performance benefits.